src/core/Global.ttslua

```diff
@@ -804,14 +804,15 @@ function onClick_toggleOption(_, id)
   end
 
   self.UI.setAttribute(id, "isOn", state)
-  optionPanel[id] = state
   applyOptionPanelChange(id, state)
 end
 
 -- sets the option panel to the correct state (corresponding to 'optionPanel')
 function updateOptionPanelState()
   for id, enabled in pairs(optionPanel) do
-    if enabled then
+    if (type(enabled) == "boolean" and enabled) or
+    (type(enabled) == "string" and enabled) or
```

Comment:
Formatting nit: indent the continuation of the previous lines (2x standard indent is common)

Reply:
fixed